locale/de_DE.UTF-8/LC_MESSAGES/palma.po

```diff
@@ -131,6 +131,18 @@ msgstr "Hilfe + Extras"
 msgid "Team display"
 msgstr "Team-Monitor"
 
+#: index.php:1090
```

Comment:
These additions were made in one language only for developing purposes. Up to now they were not added, because they had not "officially" been approved yet.